### PR TITLE
Point healthcheck links directly at the heroku app

### PR DESCRIPTION
**Pull request by TheoLeanse**
@matthew-andrews

### CODE REVIEW

File: src/server/checks/nightwatch.check.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ class NightWatchCheck extends Check {
 	setAsFailed(failures, sauceLabsUrl){
 		this.status = status.FAILED;
 		this.checkOutput = `Failed: ${failures.join(', ')}`;
-		this.panicGuide = `Saucelabs url: ${sauceLabsUrl}.  Re-run tests: https://next.ft.com/__health/rerun-nightwatch/${this.tag}`;
+		this.panicGuide = `Saucelabs url: ${sauceLabsUrl}.  Re-run tests: https://ft-next-health-eu.herokuapp.com/__health/rerun-nightwatch/${this.tag}`;
```
Comment (matthew-andrews) [inline src/server/checks/nightwatch.check.js]: `https://ft-next-health-${REGION}.herokuapp.com`

Comment (TheoLeanse) [inline src/server/checks/nightwatch.check.js]: aha cool

Comment (matthew-andrews) [inline src/server/checks/nightwatch.check.js]: can't quite see this change sorry… i think i wasn't clear, let's chat when you're back :)

Comment (TheoLeanse) [inline src/server/checks/nightwatch.check.js]: Hadn't pushed it up but it should be there now

**Comment by matthew-andrews** [comment]
+1

**Pull request merged**